packages/steamtinkerlaunch-git/steamtinkerlaunch-git.pacscript

```diff
@@ -0,0 +1,44 @@
+name="steamtinkerlaunch-git"
+url="https://github.com/frostworx/steamtinkerlaunch.git"
+build_depends="vim gcc"
+depends="git make gawk xdotool wget vim xdotool x11-utils"
+breaks="steamtinkerlaunch"
+replace="steamtinkerlaunch"
+gives="steamtinkerlaunch"
+description="Linux wrapper tool for use with the Steam client for custom launch options and 3rd party programs."
+optdepends=("strace: system call tracer"
+	"gamemode: linux gaming optimizer"
+	"mangohud: vulkan/opengl monitoring hud"
+	"winetricks: wine settings tweaker"
+	"vkbasalt: vulkan graphics enhancer"
+	"goverlay: vulkan overlay configurator"
+	"scummvm: VM for point-and-click games"
+	"gameconqueror: memory editing for cheating")
```

Comment:
🤨

Reply:
gameconqueror was in the optional dependencies provided by the stl installation guide https://github.com/frostworx/steamtinkerlaunch/wiki/Installation